local-tests/setup/tinny-environment.ts

```diff
@@ -460,6 +460,9 @@ export class TinnyEnvironment {
       await tx.wait();
     } catch (e) {
       throw new Error(`Failed to send funds to ${walletAddress}: ${e}`);
+    } finally {
+      // @ts-expect-error We don't have a user, but this works
+      this.releasePrivateKeyFromUser({ privateKey });
```

Comment:
Fixes the infinite hang during test run when you only have a couple of private keys in scope in your environment var